argoverse/evaluation/eval_detection.py

```diff
@@ -0,0 +1,412 @@
+# <Copyright 2020, Argo AI, LLC. Released under the MIT license.>
+"""Detection evaluation for the Argoverse detection leaderboard.
+
+Evaluation:
+
+    Precision/Recall
+
+        1. Average Precision: Standard VOC-style average precision calculation
+            except a true positive requires a bird's eye center distance of less
+            than a predefined threshold.
+
+    True Positive Errors
+
+        All true positive errors -- as the name implies -- accumulate error solely
+        when an object is a true positive match to a ground truth detection. The matching
+        criterion is represented by `tp_thresh` in the DetectionCfg class.
+
+        1. Average Translation Error: The average Euclidean distance (center-based) between a
+            detection and its ground truth assignment.
+
```

Comment:
i might remove the blank lines between 1, 2, and 3 here, as a minor nitpick, to reduce the length of the docstring